crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -860,16 +861,15 @@ Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and n
 nrf_cc3xx_platform - 0.9.5
 **************************
 
-Added correct TRNG categorization values for nRF5340 devices.
-
 Added
 =====
+* Correct TRNG categorization values for nRF5340 devices.
```

Comment:
Need space before the list/bullet point?
Btw, should it be a bullet point? Since the next entry is not. Or should both be?

Reply:
Yes, space needed.
I noticed I should have had all instances of "A new build of..." as a bullet point, but that would have had a snowball effect and all stuff below that would need to be moved a bit. And there is a lot of stuff under such line in all files. So, I was simply lazy and left it as a standalone line without a bullet.

Reply:
Yes, this is a bit tricky as we have the heading "Added" and everything that comes after that would be just a fragment. Anyway, I would like not to repeat the word added there.